### PR TITLE
2055 Versions Release Notes and Citations

### LINKED ISSUE #2055: Update MI Version References to 2.7
### Desired Behaviour

Update version references of Mantid Imaging to 2.7.0 as part of 2.7.0 release #2054 

### Current Behaviour

MI versions are currently set as 2.6.0

### PR DESCRIPTION
**Pull request by JackEAllen**
### Issue

Closes #2055 

### Description

1. Update Versions to 2.7
2. Update Release Notes
3. Update Citations

### Testing 

- Check all locations versions are mentioned are updated
- Check all locations contributors are mentioned are updated
- Check Citations are accurate

### Acceptance Criteria 

- [ ] All locations versions are mentioned are updated
- [ ] All locations contributors are mentioned are updated
- [ ] Citations are accurate

### Documentation

*How have you changed the documentation to reflect your changes? All changes should be noted in the appropriate file in docs/release_notes*

NA - Fresh release notes generated

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -70,8 +70,8 @@
 # General information about the project.
 project = 'MantidImaging'
 copyright = '2017-2023, Mantid Project'
```
Comment (MikeSullivan7) [inline docs/conf.py:72]: Should the copyright not be 2017-2024 now?

**Comment by samtygier-stfc** [comment]
Also, its probably time set the author ordering to be alphabetic by surname. Lots of us have made significant contributions now.

**Comment by JackEAllen** [comment]
> Also, its probably time set the author ordering to be alphabetic by surname. Lots of us have made significant contributions now.

I've reorganised to alphabetical order by Surname

**Comment by MikeSullivan7** [comment]
All looks good to me!

**Review by MikeSullivan7 — APPROVED** [review]

**Review by samtygier-stfc — APPROVED** [review]

**Comment by MikeSullivan7** [comment]
We can add your grumpiness into the Release Notes 😆

**Comment by JackEAllen** [comment]
> Grumpy about the formatting churn, but yes, lets get it merged. :-)

what formatting do you want changing? only lines 53-56 change quotations now

**Comment by samtygier-stfc** [comment]
> what formatting do you want changing? only lines 53-56 in `docs/conf.py` change quotations now

There was line 89 and 118 as well. But don't worry, its merged now.

**Pull request merged**